Tests/NavigationTests/Helpers/DistributedNavigationDelegateTestsHelpers.swift

```diff
@@ -374,12 +375,12 @@ extension DistributedNavigationDelegateTestsBase {
         }
     }
 
-    func frame(_ frameID: UInt64, _ url: URL, secOrigin: SecurityOrigin? = nil) -> FrameInfo {
+    func frame(_ frameID: UInt64!, _ url: URL, secOrigin: SecurityOrigin? = nil) -> FrameInfo {
```

Comment:
Why do we need the `!` operator here?

Reply:
to be able to pass nil when frame id flag is disabled